bootstrap/kubeadm/internal/bottlerocket/bootstrap.go

```diff
@@ -88,16 +91,19 @@ trusted=true
 	// to "public.ecr.aws" rather than the mirror's endpoint
 	// TODO: Once the bottlerocket fixes are in we need to remove the "public.ecr.aws" creds
```

Comment:
So I discussed this with the team. Looks like the issue is still open https://github.com/bottlerocket-os/bottlerocket/issues/2677

Regarding setting creds for all reg, I am not sure if its actually necessary but we suspected it might need it. I can check again 👍

Reply:
Just confirmed. It only needs public ecr as an extra entry for creds. I can change that up.